ui-tests/cypress/support/restAPI/assetsAPI.js

```diff
@@ -33,6 +33,16 @@ class AssetsController {
       )
       .then(response => ({ controller: this, response: JSON.parse(response) }));
   }
+  
+  getAssetsList(type = 'image') {
+    return cy.request({
+      url: `${controller}?type=${type}&sort=createdAt&direction=DESC&page=1&pageSize=10`,
```

Comment:
you can use _qs_ argument to define the query parameters (see [Cypress request](https://docs.cypress.io/api/commands/request#Arguments))

Reply:
done